.github/workflows/ci.yml

```diff
@@ -3,6 +3,21 @@ name: CI
 on: [push, pull_request]
 
 jobs:
+  spelling:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Set up Node
+        env:
+          DISABLE_V8_COMPILE_CACHE: "1"
+        uses: actions/setup-node@v4
+        with:
+          node-version: "20.9"
+          cache: "yarn"
+
+      - name: Run cspell
+        run: yarn global add cspell && cspell
```

Comment:
TODO: how should we set this up locally in `dev.yml`? I'm hesitant about introducing a `package.json` for just this one thing.